lib/app/features/wallet/views/pages/wallet_page/wallet_page.dart

```diff
@@ -32,28 +37,26 @@ class WalletPage extends HookConsumerWidget {
     useScrollTopOnTabPress(context, scrollController: scrollController);
 
     final activeTab = useState<WalletTabType>(WalletTabType.coins);
-    final coinsState = ref.watch(filteredWalletCoinsProvider);
-    final nftsState = ref.watch(filteredWalletNftsProvider);
-
-    final isLoading = coinsState.isLoading || nftsState.isLoading;
+    final coinsState = ref.watch(filteredCoinsProvider);
+    final nftsState = ref.watch(filteredNftsProvider);
+    final nftLayoutType = ref.watch(nftLayoutTypeSelectorProvider);
 
     List<Widget> getActiveTabContent() {
-      if (isLoading) {
-        return [
-          ListItemsLoadingState(
-            itemsCount: 7,
-            separatorHeight: 12.0.s,
-            listItemsLoadingStateType: ListItemsLoadingStateType.scrollView,
-          ),
-        ];
-      }
-
       if (activeTab.value == WalletTabType.coins) {
+        if (coinsState.isLoading) {
+          return [const _ListLoader()];
+        }
         return [
           const CoinsTab(),
           const CoinsTabFooter(),
         ];
       } else {
```

Comment:
Also, looks like there is no need in this `else` condition, since each case in the `if` above use `return`. Correct me if I wrong.